plugins/UsersManager/lang/en.json

```
@@ -99,6 +99,8 @@
         "RemoveUserAccess": "Remove access for '%1$s' for %2$s.",
         "ReportDateToLoadByDefault": "Report date to load by default",
         "ReportToLoadByDefault": "Report to load by default",
+        "SettingEnableInactiveUsersNotifications": "Receive monthly security email with list of inactive users",
```

Would using `Send` instead of `Receive` suit better here? Technically seen does Matomo send the mail, not receive it. 
```suggestion
        "SettingEnableInactiveUsersNotifications": "Send monthly security email with list of inactive users",
```

---

Replaced